software_attributes/database/owens.yaml

```diff
@@ -0,0 +1,110 @@
+# General
+github_account: sandialabs
+github_repo: OWENS.jl
+primary_uses:
+- simulation
+- optimization
+active_funding: true
+development_status: new_features
+programming_language: julia
+license_type: lgpl-3.0
+data_collection_methods:
+- modeling
+last_release_date: "2024-04-10"
+
+# Modeling
+scope:
+- turbine
+- farm
+system_type:
+- vawt
+system_location:
+- any
+time_domain: 
+- dynamic
+- steady
+turbine_model:
+- multibody_bem
+- vortex_method
+wake_model:
+- vortex_method
+loads: true
+design_optimization: true
+costs: true
+tags:
+- sectional property preprocessing
+- mass based cost
+- jointed multi-body
+- multi-turbine
+- nonlinear or linear time domain simulation
+- steady state simulation
+- hydrokinetic fluid corrections
+- linearization and export to pseudal spectral methods
+- automated campbell diagram generation
+- composite level stress
+- ultimate failure checks
+- buckling checks
+- rainflow counting
+- miner's rule fatigue acumulation
+- automatic gradients
+- VTK outputs
+
+# Distribution
+supported_platforms:
+- any
+package_manager:
+- julia
+package_manager_url:
+- git@github.com:sandialabs/OWENS.jl.git
```

Comment:
I'm not familiar with Julia's ecosystem. This is meant to be the URL to the package manager listing. For Python projects, this would be a URL on PyPI. Is the URL you've set accurate correct based on this description?

Reply:
The short answer, in context of the others, is yes.  The long answer is that OWENS is still in the process of becoming a registered julia package, and until then it can be installed using the julia built in package manager and the git url.

Reply:
Ah great, ok.